flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/AsyncVectorSearchFunction.java

```diff
@@ -49,7 +49,7 @@ public void eval(CompletableFuture<Collection<RowData>> future, Object... args)
         int topK = (int) args[0];
         GenericRowData argsData = new GenericRowData(args.length - 1);
         for (int i = 1; i < args.length; ++i) {
-            argsData.setField(i, args[i]);
+            argsData.setField(i - 1, args[i]);
```

Comment:
does this change mean that the previous code did not work? Is the symptom of this failure included in the Jira?

Reply:
We haven't added async vector search operator yet. 0.0